sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/cryptography/CryptographyAsyncClient.java

```diff
@@ -685,7 +603,7 @@ private boolean ensureValidKeyAvailable() {
         boolean keyAvailableLocally = true;
         if (this.key == null) {
             try {
-                this.key = getKey().block().getKeyMaterial();
+                this.key = getKey().block().getKey();
```

Comment:
getKey().block().getKey() ?

Reply:
this is internal.
first getKey is the KeyVaultKey, refactored the code to make it look readable.